### PR TITLE
Address review comments by @kieun

### PR DESCRIPTION
**Pull request by emlun**
Review comments by @kieun in #831

***
<a href="https://pr-preview.s3.amazonaws.com/emlun/webauthn/pull/832.html" title="Last updated on Mar 12, 2018, 12:11 PM GMT (4d42146)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/832/716a169...emlun:4d42146.html" title="Last updated on Mar 12, 2018, 12:11 PM GMT (4d42146)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -123,7 +123,7 @@ spec: WHATWG HTML; urlPrefix: https://html.spec.whatwg.org/
 
 spec: FIDO-CTAP; urlPrefix: https://fidoalliance.org/specs/fido-v2.0-ps-20170927/fido-client-to-authenticator-protocol-v2.0-ps-20170927.html
     type: dfn
-        text: CTAP2 canonical CBOR encoding form; url: ctap2-canonical-cbor-encoding-form
+        text: CTAP2 canonical CBOR encoding form; url: message-encoding
```
Comment (equalsJeffH) [inline index.bs]: the "ctap2-canonical-cbor-encoding-form" URL frag will be correct once the latest CTAP spec is published, whereupon the urlPrefix will be incorrect :-/

the fido-client-to-authenticator-protocol-v2.0-ps-20170927.html#message-encoding section does not correctly define "CTAP2 canonical CBOR encoding form" :(

It seems there is not good solution here until there the revised CTAP spec containing the \<dfn> for "CTAP2 canonical CBOR encoding form" is published.

File: index.bs [hunk 2]
```diff
@@ -323,7 +323,7 @@ below and in [[#index-defined-elsewhere]].
 : FIDO AppID
 :: The algorithms for [=determining the FacetID of a calling application=] and
     [=determining if a caller's FacetID is authorized for an AppID=] (used only in
-    the `appid` extension) are defined by [[!FIDO-APPID]].
+    the <code>[[#sctn-appid-extension|appid]]</code> extension) are defined by [[!FIDO-APPID]].
```
Comment (equalsJeffH) [inline index.bs]: I would \<dfn> "AppID" in the title for #sctn-appid-extension, and then just use [=AppID=] here.

File: index.bs (hunk 1 above)
Comment (emlun) [reply index.bs]: Ah, I see. I'll revert this one, then. Good catch!

**Comment by emlun** [comment]
Re: https://github.com/w3c/webauthn/pull/832#discussion_r173599568
Hm. I did this, and it looks a bit weird to me, but if you're happy with the result I can be too. But perhaps the `<dfn>` should be in place of the `|AppID|` in the extension description instead of in the secion heading?

**Comment by equalsJeffH** [comment]
@emlun 
> I did this, and it looks a bit weird to me
well, doing it this way the page ends up with the section title at top of frame -- i think there's at least one other section header we've done this with but I'd have to nose thru the spec source....

**Pull request merged**